osquery/tables/system/linux/deb_packages.cpp

```diff
@@ -142,5 +142,89 @@ QueryData genDebPackages(QueryContext& context) {
     return genDebPackagesImpl(context, logger);
   }
 }
+
+void genDebPackageFiles(RowYield& yield, QueryContext& context) {
+  std::vector<std::string> admindir_list{};
+  std::vector<ErrorLog> errorLogBuffer{};
+
+  if (context.hasConstraint("admindir", EQUALS)) {
+    for (const auto& admindir :
+         context.constraints["admindir"].getAll(EQUALS)) {
+      admindir_list.push_back(admindir);
+    }
+  } else {
+    admindir_list.push_back(kAdminDir);
+  }
+
+  auto dropper = DropPrivileges::get();
+  dropper->dropTo("nobody");
```

Comment:
Missing opening code block `{` before this line so that the privileges are restored before running the `GLOGLogger logger;` code (see how it's used in `deb_packages`).

Reply:
Great catch, thank you.